docs/sections/calculations/errors.md

```diff
@@ -1,302 +1,61 @@
 (calculations-errors)=
 # Dealing with errors
 
-In this section we we will intentionally introduce some bogus input parameters in order to show how to 'manually' debug problems when encountering errors.
-Workflows, which you'll see later in this tutorial, can help you avoid these issues systematically.
+In this section we will intentionally introduce some bad input parameters when setting up our calculation.
+This will allow us to illustrate how to 'manually' debug problems that might arise while managing your computations with AiiDA.
+You will learn where to look for the information of the errors and the steps you need to take to correct the issue.
+In other tutorial sections you can then learn how to systematize this error handling when designing complex workflows.
```

Comment:
Yeah, since we are "modularizing" the whole tutorial structure it was no longer clear to me if there should be an "expected order" but I guess it makes sense that they will do that afterwards.

Reply:
Once the corresponding module is written, we can simply refer to it. ;)